docs/operations/cx-server-operations-guide.md

```diff
@@ -181,16 +203,18 @@ You can do it by executing below commands.
 ```
 
 #### Plugins
-All the plugins that are required to run the SAP S/4HANA Cloud SDK Continuous Delivery Pipeline and the Piper steps
-are already pre-installed. If you update or downgrade them to a specific version, it will be lost every time the `cx-server` image is updated. 
+
```

Comment:
Does not related to the topic of the PR (Document release process)  - like also all the newlines. Blurs the focus of the PR and makes it harder to read.